### PR TITLE
validation loss miscalculation

### PR DESCRIPTION
**Pull request by satvik-venkatesh**
Thanks for this amazing project!

Currently, the val loss is calculated between estimation and low_quality. But it should be between estimation and sp_target. Could you please confirm?

It looks correct in the gsr_voicefixer.py

### CODE REVIEW

**Comment by haoheliu** [comment]
Oh ncie! Thanks for that!
